### ISSUE TITLE
Any examples or docs on how to use this? 

### ISSUE THREAD

**Issue by mmiscool** [tab58/assemble2d#1]

**Comment by tab58** [comment]
Sorry, at present this is just a stub repo. I need to update the README to reflect this. While I was building this, I realized there are several math functions I would need to port from LAPACK in order to make this work (functions to calculate matrix rank, decomposition, etc.).

**Comment by dinther** [comment]
Hi tab58, I am keen to try your solver for a code based sketch module to be used in JSCAD
any idea about time lines?

**Comment by Arredatore** [comment]
That will be absolutely great!

**Comment by mmiscool** [comment]
@Arredatore @dinther You might want to take a look at 
https://github.com/xibyte/jsketcher

**Comment by Arredatore** [comment]
> @Arredatore @dinther You might want to take a look at
> https://github.com/xibyte/jsketcher

Thank you, I know about this project. What I really need is a JS library. Not so many of them, by the way, so maybe yours even not what I'm looking for but it's hard to appreciate it.

**Comment by dinther** [comment]
> @Arredatore @dinther You might want to take a look at
> https://github.com/xibyte/jsketcher

Yeah, I had a play with it a while back. Very impressive work. I'll have a shot to see if I can make sense of all those fandangle libraries and frameworks.
